tests/test_paradiag.py

```diff
@@ -6,6 +6,194 @@
 from operator import mul
 
 
+@pytest.mark.parallel(nprocs=4)
+def test_Nitsche_BCs():
```

Comment:
Thanks for adding the tests again! I'm not sure why I removed them in the first place.